### PR TITLE
Feature user profile

### PR DESCRIPTION
**Pull request by Ashwin-op**

### CODE REVIEW

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html [hunk 1]
```diff
@@ -26,6 +26,15 @@
       <div *ngIf="showPlaceholder" class="placeholder-text d-flex justify-content-center align-items-center">
         Quickly search your favorite topics, labs by industry experts or find an event you missed
       </div>
+      <div *ngIf="showLoading" class="loading-text d-flex justify-content-center align-items-center">
+        <div class="spinner">
+          <div class="rect1"></div>
+          <div class="rect2"></div>
+          <div class="rect3"></div>
+          <div class="rect4"></div>
+          <div class="rect5"></div>
```
Comment (arpansac) [inline projects/commudle-admin/src/app/components/search-bar/search-bar.component.html:35]: Instead of different classes to each of these, we can use nth-child pseudo selector in css

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss [hunk 2]
```diff
@@ -17,27 +17,44 @@
...
+    }
+
+    nb-card-body {
+      padding: 0;
+
+      .user-image {
+        min-height: 3.5rem;
+        min-width: 3.5rem;
+        max-height: 3.5rem;
+        max-width: 3.5rem;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss:31]: Is it possible that we change this rem to px?

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss [hunk 3]
```diff
@@ -17,27 +17,44 @@
...
+        border-radius: 50%;
+        overflow: hidden;
+
+        img {
+          width: 100%;
+          height: inherit;
+          object-fit: cover;
+          object-position: center;
```
Comment (arpansac) [inline projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss:39]: Why is this needed? Asking again :P
Let's discuss, would love to understand.

File: projects/commudle-admin/src/app/components/search-bar/search-bar.component.html (hunk 1 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/components/search-bar/search-bar.component.html:35]: Done!

File: projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss (hunk 2 above)
Comment (Ashwin-op) [reply projects/commudle-admin/src/app/feature-modules/labs/components/lab/lab-details/lab-details.component.scss:31]: Done!
